apps/total-typescript/src/components/exercise-navigator.tsx

```diff
@@ -97,62 +98,22 @@ const ExerciseNavigator: React.FC<{
                         onClick={() => {
                           track(`clicked exercise in navigator`, {
                             module: module.slug.current,
-                            lesson: exercise.slug.current,
+                            lesson: exercise.slug,
                             location: router.query.lesson,
                             moduleType: module.moduleType,
                             lessonType: exercise._type,
                           })
                         }}
                       >
-                        Exercise
+                        Problem
```

Comment:
for the labels INSIDE I prefer problem/solution where an Exercise has a problem/solution even though structurally the problem IS the exercise.